embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -683,12 +693,37 @@ public boolean accept(File file) {
     protected void addDriverJarToClasspath(String glob)
     {
         // TODO match glob
-        PluginClassLoader loader = (PluginClassLoader) getClass().getClassLoader();
+        final ClassLoader loader = getClass().getClassLoader();
+        if (!(loader instanceof URLClassLoader)) {
+            throw new RuntimeException("Plugin is not loaded by URLClassLoader unexpectedly.");
+        }
+        if (loader.getClass().getCanonicalName().equals("org.embulk.plugin.PluginClassLoader")) {
```

Comment:
the old code expected `PluginClassLoader`, but the new one is not, is it intended?

Reply:
Ugh, it should be `if (!(loader.getClass()...))`. Thanks. Will fix it.

Reply:
Fixed at https://github.com/embulk/embulk-input-jdbc/pull/207/commits/bc8ebcb911f6dd3d817521b04142548f5fe5d228#diff-052bbb667d3dda11eadb326e2a45faedac30c4ddf70e5492368d6205ad274a5cR700